File: DAM 25-26/DesarolloDeInterfaces/Back/SwapShop/src/main/java/com/example/SwapShop/dto/EstadisticasPrendaDTO.java
package com.example.SwapShop.dto;

import com.example.SwapShop.modelos.Categoria;
import com.example.SwapShop.modelos.Condicion;
import com.example.SwapShop.modelos.Estilo;
import com.example.SwapShop.modelos.Talla;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class EstadisticasPrendaDTO {

    private Integer id;
    private String nombrePrenda;
    private String descripcion;
    private Talla talla;
    private Categoria categoria;
    private Condicion condicion;
    private Boolean disponible;
    private Long totalIntercambios;
}

File: DAM25-26/DesarolloDeInterfaces/Frontend/src/app/modelos/EstadisticasPrendaDTO.ts
import {EstadisticasUsuarioDTO} from "./EstadisticasUsuarioDTO";

export interface EstadisticasPrendaDTO {
  id?: number;
  nombrePrenda?: string;
  descripcion?: string;
  talla?: string;
  categoria?: string;
  estilos?: string;
  condicion?: string;
  disponible?: boolean;
  totalIntercambios?: number;
}
